vl-convert-python/vl_convert.pyi

```diff
@@ -0,0 +1,801 @@
+    ...
+
+def svg_to_jpeg(svg: str, scale: float, quality: int | None) -> bytes:
+    """
+    Convert an SVG image string to JPEG image data.
+
+    Parameters
+    ----------
+    svg
+        SVG image string
+    scale
+        Image scale factor (default 1.0)
+    quality
+        JPEG Quality between 0 (worst) and 100 (best). Default 90
+
+    Returns
+    -------
+    JPEG image data.
+    """
+    ...
+
+def svg_to_pdf(svg: str, scale: float) -> bytes:
+    """
+    Convert an SVG image string to PDF document data.
+
+    Parameters
+    ----------
+    svg
+        SVG image string
+    scale
+        Image scale factor (default 1.0)
+
+    Returns
+    -------
+    PDF document data.
+    """
+    ...
+
+def svg_to_png(svg: str, scale: float, ppi: float | None) -> bytes:
+    """
+    Convert an SVG image string to PNG image data.
+
+    Parameters
+    ----------
+    svg
+        SVG image string
+    scale
+        Image scale factor (default 1.0)
+    ppi
+        Pixels per inch (default 72)
+
+    Returns
+    -------
+    PNG image data.
+    """
+    ...
+
+def vega_to_html(
+    vg_spec: VlSpec,
+    bundle: bool | None,
+    format_locale: FormatLocale | None,
+    time_format_locale: TimeFormatLocale | None,
+    renderer: Renderer | None,
+) -> str:
+    """
+    Convert a Vega spec to a self-contained HTML document.
+
+    Parameters
+    ----------
+    vg_spec
+        Vega JSON specification string or dict
+    bundle
+        If True, bundle all dependencies in HTML file.
+        If False (default), HTML file will load dependencies from only CDN
+    format_locale
+        d3-format locale name or dictionary
+    time_format_locale
+        d3-time-format locale name or dictionary
+    renderer
+        Vega renderer. One of 'svg' (default), 'canvas',
+        or 'hybrid' (where text is svg and other marks are canvas)
+
+    Returns
+    -------
+    HTML document.
+    """
+    ...
+
+def vega_to_jpeg(
+    vg_spec: VlSpec,
+    scale: float,
+    quality: int | None,
+    allowed_base_urls: list[str] | None,
+    format_locale: FormatLocale | None,
+    time_format_locale: TimeFormatLocale | None,
+) -> bytes:
+    """
+    Convert a Vega spec to JPEG image data.
+
+    Parameters
+    ----------
+    vg_spec
+        Vega JSON specification string or dict
+    scale
+        Image scale factor (default 1.0)
+    quality
+        JPEG Quality between 0 (worst) and 100 (best). Default 90
+    allowed_base_urls
+        List of allowed base URLs for external data requests.
+        Default allows any base URL
+    format_locale
+        d3-format locale name or dictionary
+    time_format_locale
+        d3-time-format locale name or dictionary
+
+    Returns
+    -------
+    JPEG image data.
+    """
+    ...
+
+def vega_to_pdf(
+    vg_spec: VlSpec,
+    scale: float,
+    allowed_base_urls: list[str] | None,
+    format_locale: FormatLocale | None,
+    time_format_locale: TimeFormatLocale | None,
+) -> bytes:

```

Comment:
Applicable to any `None`/`Option` types:

I should've checked, but thought I recalled `rust` not having default arguments.
If this isn't the case, any `... None` should be replaced with `... None = None`

Reply:
Fixed in [`69a0549` (#185)](https://github.com/vega/vl-convert/pull/185/commits/69a0549a6665f56f6423fa1be9b4f7b048dd4765)